From a33f1a9eba489a67d69cc09dddb794aa8b9d1d35 Mon Sep 17 00:00:00 2001
From: Rui Silva <ruisil01@e121032-lin.cambridge.arm.com>
Date: Mon, 17 Dec 2018 22:47:41 +0000
Subject: [PATCH] tools: allow to override python

fix build of tools and use python given by env variable.

Signed-off-by: Rui Miguel Silva <rui.silva@linaro.org>
---
 tools/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/Makefile b/tools/Makefile
index cb1683e..9bf9ceb 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -138,7 +138,7 @@ tools/_libfdt.so: $(LIBFDT_SRCS) $(LIBFDT_SWIG)
 		CPPFLAGS="$(_hostc_flags)" OBJDIR=tools \
 		SOURCES="$(LIBFDT_SRCS) tools/libfdt.i" \
 		SWIG_OPTS="-I$(srctree)/lib/libfdt -I$(srctree)/lib" \
-		$(libfdt_tree)/pylibfdt/setup.py --quiet build_ext \
+		${PYTHON} $(libfdt_tree)/pylibfdt/setup.py --quiet build_ext \
 			--build-lib tools
 
 ifneq ($(CONFIG_MX23)$(CONFIG_MX28),)
-- 
2.7.4

